packages/mui-material-next/src/styles/CssVarsProvider.tsx

```diff
@@ -13,7 +13,8 @@ import defaultTheme from './defaultTheme';
 
 const shouldSkipGeneratingVar = (keys: string[]) =>
   !!keys[0].match(/(typography|mixins|breakpoints|direction|transitions)/) ||
-  (keys[0] === 'palette' && !!keys[1]?.match(/(mode|contrastThreshold|tonalOffset)/));
+  (keys[0] === 'palette' && !!keys[1]?.match(/(mode|contrastThreshold|tonalOffset)/)) ||
+  (keys[0] === 'motion' && !!keys[1]?.match(/(create|getAutoHeightDuration)/));
```

Comment:
```suggestion
```

The function will be skipped by default, no need to put them here.

Reply:
Ah, I didn't know that, alright then I will just not add the `motion` key anywhere :)